doc/contributing.rst

```diff
@@ -635,10 +635,11 @@ When adding or removing dependencies, please consider applying the changes in
 the following files:
 
 - ``environment.yml``
-  contains development dependencies that cannot be installed from
-  `PyPI <https://pypi.org/>`_
+  contains all the development dependencies; these are all from
+  `conda-forge <https://conda-forge.org/>`_
 - ``setup.py``
-  contains all Python dependencies, regardless of their installation source
+  contains Python dependencies related to testing, that installed from
```

Comment:
Could you restore the previous text here? The test dependencies should also be installed from environment.yml.

Reply:
done